test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3029,7 +3026,6 @@ static private void valueTypeIdentityObjectTestHelper(String testName, String su
 			superClassName = "java/lang/Object";
 		}
 		Class valueClass = ValueTypeGenerator.generateValueClass(testName, superClassName, fields, extraClassFlags);
-		assertTrue(PrimitiveClass.isPrimitiveClass(valueClass));
```

Comment:
Instead of removing the assertion, can it be replaced with `assertTrue(valueClass.isValue());`